crates/undo/src/lib.rs

```diff
@@ -222,36 +222,6 @@ impl Default for ChangeChainSettings {
     }
 }
 
-impl ChangeChain {
```

Comment:
What happened to this set of code? Couldn't figure out where it is now?

Reply:
Its nowhere. It was unused code

Reply:
Wow, didn't clippy warn it 😱